core/src/main/java/com/sequenceiq/cloudbreak/controller/validation/template/TemplateValidator.java

```diff
@@ -174,11 +186,17 @@ private void validateVolumeCount(VolumeTemplate value, VmType vmType, VolumePara
     }
 
     private void validateVolumeSize(VolumeTemplate value, VmType vmType, VolumeParameterType volumeParameterType,
-        ValidationResult.ValidationResultBuilder validationBuilder) {
+        ValidationResult.ValidationResultBuilder validationBuilder, InstanceGroup instanceGroup) {
         if (vmType != null && needToCheckVolume(volumeParameterType, value.getVolumeCount())) {
             VolumeParameterConfig config = vmType.getVolumeParameterbyVolumeParameterType(volumeParameterType);
             if (config != null) {
-                if (value.getVolumeSize() > config.maximumSize()) {
+                if (isIDBrokerInstanceGroup(instanceGroup)) {
```

Comment:
Like above, please remove the new `isIDBrokerInstanceGroup(instanceGroup)` branch and reuse the original max limit validation without any changes.